src/Util/TraversableToCollection.php

```diff
@@ -22,14 +22,16 @@
 final class TraversableToCollection
 {
     /**
-     * @param \Traversable|array $value
+     * NEXT_MAJOR: Add typehint.
+     *
+     * @param iterable<mixed, mixed> $value
      *
      * @throws \TypeError
      *
      * @return Collection<int|string, mixed>
      *
      * @phpstan-template T
-     * @phpstan-param \Traversable<T>|array<T> $value
+     * @phpstan-param iterable<mixed, T> $value
```

Comment:
The class traversableToCollection should be renamed then?

Reply:
Yes, I thought about doing it in another PR with the deprecation etc. To not have a too big PR about phpstan (and stay pedantic).

Reply:
Might be worth doing an issue about it, or atleast add a NEXT_MAJOR comment, wdyt?

Reply:
done